src/ConfirmProvider/ConfirmProvider.test.js

```diff
@@ -0,0 +1,135 @@
+import { ConfirmProvider, useConfirm } from "./ConfirmProvider";
+import { fireEvent, render } from "@testing-library/react";
+
+import { Button } from "@mui/material";
+import React from "react";
+import ThemeProvider from "../ThemeProvider";
+import { action } from "@storybook/addon-actions";
+// eslint-disable-next-line camelcase
+import { unstable_createMuiStrictModeTheme } from "@mui/material/styles";
```

Comment:
Whats going on here? Why are we using an unstable flagged feature? Might need to talk me through this one.